### ISSUE TITLE
Core feature sets

### ISSUE THREAD

**Issue by tkrajcar** [tkrajcar/new-mud-index-ideas#1]
TBD

**Comment by BlackRookSix** [comment]
I picture Moderators as team members who work within the site itself, not within the code.  Just a small point.  The development methodology presented by Github should work gloriously for this community-based approach for the development team.  I am not sure about upvoting edits, but the merges are definitely peer-reviewed by dev leads before committing to the trunk.

**Comment by tkrajcar** [comment]
@BlackRookSix Yeah, that's why I called out technical ownership and moderator groups separately - devs will use Github, mod functionality will be built into the site itself.
